internal/csi-addons/rbd/replication.go

```diff
@@ -924,6 +927,10 @@ func getLastSyncInfo(description string) (*replication.GetVolumeReplicationInfoR
 	if description == "" {
 		return nil, fmt.Errorf("empty description: %w", corerbd.ErrLastSyncTimeNotFound)
 	}
+	log.DebugLog(
+		ctx,
+		"description=%q",
+		description)
```

Comment:
description does not need to be quoted as you do now, it contains many unneeded `\` escapes. Use `"descripion: %s` instead, or maybe `"last sync details: %s", description`.

Reply:
the parameters passed to the function are not very long, they probably fit on a single line

Reply:
can you change this to a single line as mentioned above?